definition_test.go

```diff
@@ -31,27 +31,36 @@ func TestJSONSpecification(t *testing.T) {
     "node_key_pem": "` + jsonbytes(node1_1_key) + `",
     "cluster_cert_pem": "` + jsonbytes(signing1_cert) + `"
 }`)
-	cluster, err := createFromJSON(validJSON, 0, NullLogger)
-	cluster.Terminate()
+	cluster, err := createFromJSON(validJSON, 1, NullLogger)
 	if err != nil {
 		t.Fatal("Got error constructing valid cluster:", err)
 	}
+	cluster.Terminate()
```

Comment:
Terminate() here, and in a few other spots below, was causing a panic in tests because cluster was nil when an error occurred.